platform/security/common/src/main/java/org/codice/ddf/security/common/jaxrs/RestSecurity.java

```diff
@@ -93,8 +95,8 @@ public static void setUserOnClient(String username, String password, Client clie
                 }
             }
             String basicCredentials = username + ":" + password;
-            String encodedHeader = BASIC_HEADER_PREFIX + new String(
-                    Base64.encodeBase64(basicCredentials.getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
+            String encodedHeader = BASIC_HEADER_PREFIX + Base64.getEncoder()
+                    .encodeToString(basicCredentials.getBytes(StandardCharsets.UTF_8));
```

Comment:
Did you manually verify that a source can be configured with BASIC authentication and successfully log into an endpoint protected with BASIC?